news/_posts/2022-01-13-new-theme-announce.md

```diff
@@ -0,0 +1,59 @@
+---
+layout: post
+title: We are working on a new front-office theme
+subtitle: Here are some informations about it!
+date: 2022-01-13
+authors: [NeOMakinG]
+icon: icon-laptop
+tags: [news, theme, 8.0]
+---
+
+# The PrestaShop team is working on a new theme
+
+For a lot of time, the community asked us to rework the classic theme because a lot of things are missing inside it. We announced that we are working on a new front-office theme that you can find [here](https://github.com/PrestaShop/theme-refacto/compare/develop...mparvazi:auto-rtl).
```

Comment:
```suggestion
For a lot of time, the community asked us to rework the classic theme because many things were missing inside it. We announce that we are working on a new front office theme, and you can find [here](https://github.com/PrestaShop/theme-refacto/compare/develop...mparvazi:auto-rtl).
```